src/libsystemd/sd-event/sd-event.c

```diff
@@ -126,7 +126,7 @@ struct sd_event {
         LIST_HEAD(struct inode_data, inode_data_to_close);
 
         /* A list of inotify objects that already have events buffered which aren't processed yet */
-        LIST_HEAD(struct inotify_data, inotify_data_buffered);
+        LIST_HEAD(struct inotify_data, buffered_inotify_datas);
```

Comment:
I don't know if it matters, but "data" has no plural form and "datas" does not exist as an English word. The commend above this line suggests that using "objects" instead may be appropriate here.

Reply:
Interesting. But I recently heard that 'datas' is becoming approved as a plural of 'data', even though 'data' is already plural.

Reply:
"datas" would be nice, but dictionaries don't list it, so it seems that it's not widely accepted yet.
I renamed it to `buffered_inotify_data_objects` now.